packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -80,6 +88,9 @@ export const Alert: React.FunctionComponent<AlertProps & OUIAProps> = ({
         {action && (typeof action === 'object' || typeof action === 'string') && (
           <div className={css(styles.alertAction)}>{action}</div>
         )}
+        {action && typeof action === 'function' && (
+          <div className={css(styles.alertAction)}>{action({ title, variantLabel })}</div>
```

Comment:
I don't understand, couldn't this be
`<div className={css(styles.alertAction)}>{action}</div>` and since consumer is passing in `title` and `variantLabel` as props they already have knowledge of the props so aren't we just giving it back to them?